app/controllers/epp/base_controller.rb

```diff
@@ -8,7 +8,7 @@ class AuthorizationError < StandardError; end
     before_action :ensure_session_id_passed
     before_action :generate_svtrid
     before_action :latin_only
-    before_action :validate_against_schema
+    # before_action :validate_against_schema
```

Comment:
Please, remove this comment here and put this action back online.

Use skip_before_action in errors controller instead.